src/PhpSpec/ServiceContainer.php

```diff
@@ -234,7 +234,7 @@ public function configure()
      *
      * @return array
      */
-    private function getPrefixAndSid($id)
+    public function getPrefixAndSid($id)
```

Comment:
I solved this by creating identical private method `getPrefixAndSid()` in the trait.

The code is in my branch:
https://github.com/gajdaw/phpspec/tree/base-extension-trait

Because PR is closed the code is not updated here. I wasn't sure if I would be able to fix broken tests on PHP 5.3. But I did it. 

If you consider this solution valuable can you, please, reopen this PR?